### ISSUE TITLE
Onboarding experience survey

### ISSUE THREAD

**Issue by Mehonoshin** [howtohireme/give-me-poc#319]
**Value**
Onboarding experience is critically important for us. We want to have a fast and smooth process, to prevent losing the motivation among our members.

When the user has finished onboarding we want to send him an email with a link. Following the link, he opens a form on POC, that asks him several questions about his onboarding experience.

When the user finished the survey, we notify staff via email about a new response. It should be available at `Onboarding feedbacks` section.

Every 3 days since the survey was offered, we remind users with uncompleted surveys to do that.

Also, we want to display pending survey for a member at his dashboard root page.
Let's create `Dashboard::PendingTasks` model/service, that looks through the database and detects if he has some pending activities. At the moment pending survey is the only possible activity, but soon we'll have much more of them.

So basically we want to notify member, and display survey request at his dashboard.

**Possible solution**
We need to create `Developer::Onboarding::SurveyResponse` model, that will contain a specific member's answers for the survey. Answers should be kept in the field with `jsonb` type. We might want to change the questions list, and it will allow us not to change the data schema for new questions.

Questions themselves might be hardcoded inside the view.

For scheduled events, like notifications, we need to have a `clockwork` gem, that schedules some events.
